clusterDeployer.py

```diff
@@ -535,13 +535,14 @@ def _get_discovery_ign_ssh_priv_key(self, infra_env: str) -> str:
 
     def update_etc_hosts(self) -> None:
         cluster_name = self._cc.name
-        api_name = f"api.{cluster_name}.redhat.com"
+        api_names = [f"api.{cluster_name}.redhat.com", f"prometheus-k8s-openshift-monitoring.apps.{cluster_name}.redhat.com"]
         api_vip = self._ai.get_ai_cluster_info(cluster_name).api_vip
 
         hosts = Hosts()
-        hosts.remove_all_matching(name=api_name)
+        for api_name in api_names:
+            hosts.remove_all_matching(name=api_name)
```

Comment:
Unfortunately, `python_hosts` doesn't seem to have a `remove_all_matching()` method that takes a list of names.  We instead have to iterate through all names ourselves.